### DISCUSSION TITLE
Train Glow TTS for Portuguese

### DISCUSSION THREAD

**Discussion by joseluismoreira** [coqui-ai/TTS#500] [General Q&A]
I am interested in training Glow TTS / Tacotron2 DDC for Portuguese language. 

I couldnt find proper instructions about the changings I have to do for it, especially how to choose the phonemes to use in the config file. 

I would also to perform transfer learning if it is possible, as @Edresson has done in his paper (https://arxiv.org/pdf/2005.05144.pdf) to decrease the training time., but it is not a hard requirement for me.

May someone help me about what I have to do in order to train GlowTTS in Portuguese , please?

I am planning to use Common Voice. (I have also thought about MLS, but it has not punctuation)

**Comment by erogol** [comment]
maybe @Edresson  can help you as he is the most proficient person training Portuguese models using :frog: TTS.

**Reply by joseluismoreira** [reply]
Thank  you @erogol . I reached him and it is being very helpful for me.

**Comment by joseluismoreira** [comment]
@erogol and @Edresson I am trying to train a **Multispeaker Portuguese Model**. So, I am finetuning (restore_path) the  **Portuguese Single speaker Model** from this notebook (https://colab.research.google.com/drive/1sh96-f9J2BJcLm1rz5KBkwq3YP83bEr5?usp=sharing) and adapting the config for using in a multispeaker way. The speaker embeddings were generated from the mueller91's model. I am training using CommonVoice. 

The loss is decreasing, but the synthesized utterances appear to remain related with the single speaker initial model (the voice is similar to that single speaker, instead of variable across the speakers). :/ 

I am wondering if that is related with some error I am making in config. So I would like to ask for help. My questions are:

1) Does my config / experiment makes sense ?  Here is the config: https://drive.google.com/file/d/1W-WYL_CfyPKw40GVhQevVWaTXkwg2CFO/view?usp=sharing

2) I am also wondering if this finetuning is even good or maybe finetuning from the VCTK english glowtts would be better. 

Thank you!!

**Reference used:** 
**Single Speaker Portuguese Model** (drive config id 1V0eCe19GjX3qxR02wKb_epIZOlaAk-Te ;  drive id: 1FvKadj9Um7t97KttEpyuqlk9i-oChBDq ;  ) 

**Speaker Enc model** (https://drive.google.com/drive/folders/15oeBYf6Qn1edONkVLXe82MzdIi3O_9m3)
